### PR TITLE
Implement the calendar icon and date visualisation in batch edit

### PR DESCRIPTION
**Pull request by Coenego**
image 2015-01-29 at 3 27 37 pm

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

File: apps/timetable/admin/ui/css/admin.css [hunk 1]
```diff
@@ -367,6 +367,50 @@
...
+}
+
+.gh-event-calendar-icon .month {
+    font-size: 8px;
+    line-height: 10px;
+    text-transform: uppercase;
+}
+
+.gh-event-calendar-icon .day {
+    font-size: 16px;
```
Comment (bp323) [inline apps/timetable/admin/ui/css/admin.css]: I think it looks nicer when changing it to 13px, wdyt?

Comment (bp323) [inline apps/timetable/admin/ui/css/admin.css]: The design seems to have a smaller calendar icon as well.

**Comment by bp323** [comment]
Reassigning to @Coenego for followup

**Comment by Coenego** [comment]
Reassigning to @bp323 for review after follow-up

**Comment by bp323** [comment]
Verified and merged, looks great!

**Pull request merged**